music-ecom/src/pages/ProductList/ProductList.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import Navbar from "../../Components/Navbar/Navbar";
```

Comment:
You import using an absolute path instead of using a relative path.
You can check this article if you want to know about this thing.
https://dynamicprogrammer.hashnode.dev/absolute-path-in-react-js

Reply:
ok.. thnks for sharing